app/views/users/passwords/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<!--Descriptive page name, messages and instructions-->
+<h3 class="heading"><%= ts("Forgotten your password?") %></h3>
+<p>
+  <%= ts("If you've forgotten your password, we can send instructions that will allow you to reset it.") %>
```

Comment:
There's another instance of this text in the form for changing user names that should be updated to match:

https://github.com/otwcode/otwarchive/blob/5cf864c5d48637c8bea9dc9223ebceb086ca44ff/app/views/users/change_username.html.erb#L44-L52

Maybe we should link to the new password reset page instead of having a mini-form on the change username page?

Reply:
Whoops, sorry! I missed that form. (The dangers of unpolished code!)

Is that block of code reachable? I'm having trouble getting it to display.